configtx/consortiums_test.go

```diff
@@ -1486,6 +1486,60 @@ func TestRemoveConsortiumOrg(t *testing.T) {
 	gt.Expect(c.Consortium("Consortium1").Organization("Org1")).To(BeNil())
 }
 
+func TestSetConsortiumOrgModPolicy(t *testing.T) {
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+
+	consortiums, _ := baseConsortiums(t)
+
```

Comment:
nit: blank line

There are a couple of testes below with the same pattern.Basically, the setup of `t` is a chunk of code, the setup of the data that goes into config is another. We don't need to have vertical space between the lines.